locales-pending/emails-all.ftl

```diff
@@ -1,104 +0,0 @@
-# This Source Code Form is subject to the terms of the Mozilla Public
-# License, v. 2.0. If a copy of the MPL was not distributed with this
-# file, You can obtain one at http://mozilla.org/MPL/2.0/.
-
-# Strings in this file are not yet final, and thus should not be localised yet.
-
-email-footer-reason-subscriber = You’re receiving this automated email as a subscriber of { -brand-mozilla-monitor }. If you received it in error, no action is required. For more information, please visit <support-link>{ -brand-mozilla } Support</support-link>.
```

Comment:
I see this one on `main` still, so it seems like we still need it https://github.com/mozilla/blurts-server/blob/main/src/emails/components/EmailFooter.tsx#L61

More in these I suspect we still need, like the HIBP footer and support footer

Unless I'm misunderstanding locales-pending?